src/interface/src/app/types/plan.types.ts

```diff
@@ -0,0 +1,17 @@
+import { Region } from "./region.types";
+
+export interface Plan extends BasePlan {
+  id: string;
```

Comment:
question: how big can the ids get? is it safer to use string instead of number? there are no native 64-bit integers in javascript

Reply:
@elsiel23

Reply:
Great question. From reading a few Stack Overflow threads, it seems like string is the recommended solution. The max Number integer value is 9,007,199,254,740,991, which is pretty large, but better to be safe.

Reply:
Ah I see, let's keep it as string here then. :)